src/EPR.Payment.Facade.Common.UnitTests/Mappings/AutoMapperTests.cs

```diff
@@ -32,39 +33,43 @@ public void AutoMapper_Configuration_IsValid()
             configAction.Should().NotThrow();
         }
 
-        [TestMethod, AutoMoqData]
-        public void PaymentRequestDto_To_GovPayPaymentRequestDto_Mapping_IsValid(PaymentRequestDto paymentRequestDto)
+        [TestMethod]
+        public void PaymentRequestDto_To_GovPayRequestDto_Mapping_IsValid()
         {
-            var govPayPaymentRequestDto = _mapper.Map<GovPayRequestDto>(paymentRequestDto);
+            var fixture = new Fixture();
+            var paymentRequestDto = fixture.Create<PaymentRequestDto>();
+            var govPayRequestDto = _mapper.Map<GovPayRequestDto>(paymentRequestDto);
 
             using (new FluentAssertions.Execution.AssertionScope())
             {
```

Comment:
we can import the namespace;
 using FluentAssertions.Execution;
 
and then initialise AssertionScope with the class name only;
![image](https://github.com/user-attachments/assets/f6db194d-d921-4124-99b2-dca1caeda044)

Reply:
I have changed all tests and now import the namespace